core/src/main/scala/scalacss/package.scala

```diff
@@ -61,6 +71,13 @@ package object scalacss {
   case class CssKeyframesEntry(name  : KeyframeAnimationName,
                                frames: Map[KeyframeSelector, StyleStream]) extends CssEntry
 
+  case class CssFontFace(fontFamily  : String,
+                         src         : NonEmptyVector[String],
+                         fontStretch : Value,
+                         fontStyle   : Value,
+                         fontWeight  : Value,
+                         unicodeRange: UnicodeRange) extends CssEntry
```

Comment:
The types lie here. Everything after `src` should be surrounded by `Option[_]`.

`UnicodeRange.full` was a good idea but it forces a `if (fontface.unicodeRange != UnicodeRange.full)` when rendering. Best to just capture what is specified by the user and use `Option[_]` here too.

After that, if `UnicodeRange.full` isn't used anywhere else, please delete it. Versions of Unicode and many and they change and we don't want to have to keep it up-to-date.